repos/system_upgrade/el7toel8/actors/checkminimumrequirements/actor.py

```diff
@@ -0,0 +1,22 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import library
+from leapp.models import HardwareInfo, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckMinimumRequirements(Actor):
+    """
+    These actor will check the system against RHEL8 minimal hardware requirements,
+    using the following resource: https://access.redhat.com/articles/rhel-limits
+
+    Note: Disk space will not be checked with these actor, rely on dnf actor to
+    check it.
+    """
+
+    name = 'checkminimumrequirements'
+    consumes = (HardwareInfo, )
+    produces = (Report, HardwareInfo)
```

Comment:
How you can consume and provide msg in one actor? That's wrong. Where is actor scanning the machine?